pkg/names/names.go

```diff
@@ -76,6 +76,12 @@ func GenVProxyConfigMapName(vdb *vapi.VerticaDB, sc *vapi.Subcluster) types.Name
 	return GenNamespacedName(vdb, sc.GetVProxyConfigMapName(vdb))
 }
 
+// GenVProxyConfigMapNameFromVDB returns the name of the client proxy configmap from
+// subcluster name string
+func GenVProxyConfigMapNameFromVDB(vdb *vapi.VerticaDB, scName string) types.NamespacedName {
```

Comment:
Don't need this, use GenVProxyConfigMapName

Reply:
Duplicate conversation as https://github.com/vertica/vertica-kubernetes/pull/980#discussion_r1851608704